scientific/E3SM_DV3D_Native_Grid_Test/E3SM_DV3D_ne30_Native_Grid_Test.html

```diff
@@ -13172,7 +13172,7 @@ <h2 id="To-Interact-with-the-3D-image...">To Interact with the 3D image...<a cla
 <div class="text_cell_render border-box-sizing rendered_html">
 <h2 id="Get-the-tutorial-data">Get the tutorial data<a class="anchor-link" href="#Get-the-tutorial-data">&#182;</a></h2><p>This tutorial uses the E3SM model version 1, 1 degree (standard resolution (ne30)) CMIP6 data which is available on NERSC's HPSS storage system. See the <a href="https://e3sm.org/data/get-e3sm-data/released-e3sm-data/v1-1-deg-data-cmip6/v1-1-deg-cmip6-data-on-hpss/">V1 1 Deg CMIP6 Data on HPSS</a> post on the E3SM.org website to download the data.</p>
 <p>The data file for this tutorial is "20180129.DECKv1b_piControl.ne30_oEC.edison.cam.h0.0500-01.nc" from the CMIP6 standard resolution (ne30) piControl run.</p>
-<p>The grid file used is "ne30np4_latlon.091226.nc". To download this grid file, go to <a href="https://portal.nersc.gov/project/acme/hdavis/">https://portal.nersc.gov/project/acme/hdavis/</a> and click on the "ne30np4_latlon.091226.nc" file which should automatically download via your browser. To facilitate running this notebook, the CMIP6 data file can also be downloaded from this same location.</p>
+<p>The grid file used is "ne30np4_latlon.091226.nc". To download this grid file, go to <a href="https://portal.nersc.gov/project/acme/hdavis/">https://portal.nersc.gov/project/acme/hdavis/</a> and click on the "ne30np4_latlon.091226.nc" file whcih should automatically download via your browser. To facilitate running this notebook, the CMIP6 data file can also be downloaded from this same location.</p>
```

Comment:
I noticed the 'whcih' was added back in, should remain as 'which' no?

Reply:
Yes, it should be fixed - thanks!  I pushed a commit to fix the typo + some other minor edits.